app/src/main/java/org/openobservatory/ooniprobe/common/ExceptionManager.java

```diff
@@ -1,7 +1,9 @@
 package org.openobservatory.ooniprobe.common;
 
+import ly.count.android.sdk.Countly;
```

Comment:
Do I understand correctly that count.ly also works without firebase?

Reply:
the crash logging yes, it has nothing to do with firebase

Reply:
I can build the fdroid flavor on my machine.

Reply:
it is ok as I wrote above